components/serializer.rst

```diff
@@ -737,6 +737,10 @@ There are several types of normalizers available:
     :phpclass:`DateTime` and :phpclass:`DateTimeImmutable`) into strings.
     By default, it uses the `RFC3339`_ format.
 
+:class:`Symfony\\Component\\Serializer\\Normalizer\\DateTimeZoneNormalizer`
+    This normalizer converts :phpclass:`DateTimeZone` objects into strings.
+    By default, it uses the `RFC3339`_ format.
```

Comment:
Is this true? If I don't miss anything the identifiers used by PHP are not based on an RFC, right?